app/jobs/cleanup_account_job.rb

```diff
@@ -1,18 +1,27 @@
 class CleanupAccountJob < ActiveJob::Base
   def perform(account)
-    cleanup_solr(account)
-    cleanup_fedora(account)
+    account.switch do
+      cleanup_solr(account)
```

Comment:
Good point here, although if you look closely at the existing code, the fedora cleanup _currently_ sits inside the same `account.switch` (see the refactor I did in `cleanup_fedora`). I had refactored to have a single `switch` call, rather than multiple.   That said, I'm honestly not certain the fedora cleanup requires the `switch`. I'll dig in and see which ones require it and which do not.